<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smoothFilterPropForm</class>
 <widget class="QWidget" name="smoothFilterPropForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smooth Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="cellToPoint_radioButton">
     <property name="text">
      <string>Cell Interpolation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="meshSmooth_radioButton">
     <property name="text">
      <string>Mesh Smooth</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="smooth_groupBox">
     <property name="title">
      <string>mesh smooth</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>smooth coefficient</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="coef_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
